Package.swift

```diff
@@ -16,11 +16,17 @@ let package = Package(
       name: "SwiftOpenAI",
       targets: ["SwiftOpenAI"]),
   ],
+  dependencies: [
+    .package(url: "https://github.com/swift-server/async-http-client.git", from: "1.25.2"),
+  ],
```

Comment:
I'm not sure this is possible, but can we scope this to only linux for now? ie wrap in `#if os(Linux)` ?

Reply:
I've made this change by using `condition: .when(platforms: [.linux])`, since `#if os(Linux)` doesn't work in Package.swift.